src/makeCalculator.js

```diff
@@ -3,8 +3,35 @@
 /**
  * @return {object}
  */
+
+
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+        return a - b;
+    },
+    multiply(a, b) {
+      return a * b;
+    },
+    divide(a, b) {
+      return a / b;
+    },
+    operate(callback, number) {
+      this.result = callback(this.result, number);
```

Comment:
[CODE STYLE] - Instead of reassigning the result directly, use arithmetic operators with assignment operator `=` to modify the existing value. This will make your code cleaner and more in line with the arithmetic operation being performed. For example, `this.result += number` for the `add` method.